src/api.rs

```diff
@@ -10,6 +10,34 @@ use crate::helpers;
 use crate::responses::*;
 use storage_proofs::sector::SectorId;
 
+#[cfg(not(target_os = "windows"))]
+pub unsafe fn raw_to_file(raw: *mut libc::c_void) -> std::fs::File {
+    use std::os::unix::io::{FromRawFd, RawFd};
+
+    std::fs::File::from_raw_fd(raw as RawFd)
+}
+
+#[cfg(not(target_os = "windows"))]
+pub unsafe fn file_to_raw(file: std::fs::File) -> std::os::unix::io::RawFd {
+    use std::os::unix::io::IntoRawFd;
+
+    file.into_raw_fd()
+}
+
+#[cfg(target_os = "windows")]
+pub unsafe fn raw_to_file(raw: *mut libc::c_void) -> std::fs::File {
```

Comment:
BLOCKING: What is the behavior of this function when given a file descriptor to any of the non-`File` in [this list](https://doc.rust-lang.org/std/os/unix/io/trait.FromRawFd.html#implementors)? If it's undefined behavior, you should say so in function comments.